vscode-extension/src/aiConfigEditor.ts

```diff
@@ -124,6 +124,55 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
 
     let isWebviewDisposed = false;
 
+    function updateServerWithRetry(
+      serverUrl: string,
+      document: vscode.TextDocument
+    ) {
+      updateServerState(serverUrl, document)
+        .then(() => {
+          // In case of previous failure, reset to editable state
+          webviewPanel.webview.postMessage({
+            type: "set_readonly_state",
+            isReadOnly: false,
+          });
+        })
+        .catch((e) => {
+          if (isWebviewDisposed) {
+            // Ignore errors caused by closing the webview while the server update
+            // request is in flight (e.g. closing config w/ unsaved changes will
+            // emit onDidChangeTextDocument with reverted unsaved changes)
+            // See #1201 for full details.
+            console.info(

```

Comment:
Just checking if this should be piped to the output channel or if this is just a console log for debugging

Reply:
This is mainly for debugging, I don't think this would be useful for users to see